code401challenges/src/main/java/code401challenges/utilities/Animal.java

```diff
@@ -0,0 +1,5 @@
+package code401challenges.utilities;
+
+public interface Animal {
+    public void sound();
```

Comment:
Another fix for your test would be to add a getName method to the animal interface, then you can test both `Cat` and `Dog` instances as type `Animal`